### PR TITLE
Add Swiggy

### LINKED ISSUE #3951: Request: Swiggy Icon
**Name:** Swiggy
**Website:** https://www.swiggy.com/
**Alexa rank:** https://www.alexa.com/siteinfo/swiggy.com
  

**Official resources for icon and color:** https://www.swiggy.com/

### PR DESCRIPTION
**Pull request by adamalston**
swiggy

Closes #3951 

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Source is the company that created the icon

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -7205,6 +7205,11 @@
             "hex": "FA7343",
             "source": "https://developer.apple.com/develop/"
         },
+        {
+            "title": "Swiggy",
+            "hex": "FC8019",
+            "source": "https://oppositehq.com/portfolio/swiggy-brand-identity-design/"
```
Comment (adamrusted) [inline _data/simple-icons.json:7211]: Could you update the source to be the [brand's website](https://www.swiggy.com/) - given the SVG is available there - and is likely to better reflect any changes in the future.

File: icons/swiggy.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...wiggy icon</title><path d="M13.05 4.7123a.3564.3564 0 01.3765.402c.0024.1103.0042.2206.0043.331.0007.7109-.003 1.422.0025 2.1328.0039.5203.0954.6205.5993.683a17.7998 17.7998 0 003.778-.055 5.9386 5.9386 0 001.8875-.4486.6087.6087 0 00.4136-.723 8.1981 8.1981 0 00-6.4124-6.854 7.9307 7.9307 0 00-4.3786.2917 8.0624 8.0624 0 00-3.2325 2.1022 7.5272 7.5272 0 00-2.2165 5.142 13.4266 13.4266 0 001.8309 6.622 1.6107 1.6107 0 001.596.8967c.784-.0188 1.569-.0042 2.3535-.0042v-.0042q1.2504 0 2.5006.0006c.2992.0007.536.0722.5336.4468-.0055.8703.0008 1.7407-.004 2.611-.0015.2641-.0759.5184-.3914.5209-.3167.0025-.3917-.2508-.3957-.5155-.006-.4166.0003-.8335.0008-1.2502.0005-.5413-.0882-.6682-.628-.7669a9.2436 9.2436 0 00-2.8178-.0239 3.1154 3.1154 0 00-.9608.2338c-.2542.1196-.3553.294-.2318.5608.1285.2775.2483.5607.3999.8255a52.655 52.655 0 004.1122 5.9474c.1847.2393.3278.2443.5173.0126.4185-.5118.858-1.0064 1.272-1.5216a41.0756 41.0756 0 004.2921-6.3125 17.0071 17.0071 0 001.7936-4.6462 1.444 1.444 0 00-1.2141-1.962 8.332 8.332 0 00-2.0048-.194c-1.0909-.008-2.182-.0041-3.2729-.0031-.2938.0003-.5123-.0841-.5125-.4288-.0006-1.2135-.0022-2.4271.0022-3.6406.001-.259.1259-.4181.4073-.4085z"/></svg>
```
Comment (adamrusted) [inline icons/swiggy.svg:1]: There appear to be some differences between this version, and the one embedded in the website header. Any chance you could look to rebuild to 3 decimals from the new source? 

image

**Review by adamrusted — CHANGES_REQUESTED** [review]
Thanks @adamalston!
I've mentioned it below, but it's always worth linking directly to the brand as opposed to an agency that created the logo on their behalf. Should they rebrand and go with a different agency, we're less likely to see a change on the agency's site.

File: icons/swiggy.svg (hunk 2 above)
Comment (adamalston) [reply icons/swiggy.svg:1]: Tried redoing the entire process of making the icon. I opened Inkscape and set a clip to 'crop' the svg. Do you know how I could remove the transform attribute in Inkscape:
```<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Swiggy icon</title><path d="M8.137 24a.801.801 0 01-.05-.041 63.638 63.638 0 01-3.9-5.472c-.546-.916-.9-1.63-.833-1.814.179-.48 3.355-.743 4.333-.308.298.132.29.307.29.409 0 .44-.021 1.619-.021 1.619a.442.442 0 10.883-.002l-.005-2.939c0-.255-.28-.318-.332-.328-.51-.003-1.547-.006-2.66-.006-2.457 0-3.006.1-3.424-.172-.904-.592-2.383-4.577-2.417-6.82-.05-3.162 1.825-5.901 4.464-7.258A8.126 8.126 0 018.13 0c4.177 0 7.617 3.153 8.075 7.21l.001.01c.084.98-5.321 1.19-6.39.904-.164-.044-.206-.212-.206-.284l-.006-2.844a.443.443 0 00-.885.002l.009 3.865a.33.33 0 00.268.32h3.354c1.79 0 2.541.207 3.042.588.332.253.461.739.349 1.37C14.736 16.755 8.376 23.71 8.137 24zm17.248-13.102c1.605.687 2.599 1.45 2.599 3.125 0 1.71-1.296 2.816-3.301 2.816-1.625 0-2.929-.736-3.579-2.02L21 14.612l1.91-1.107.126.205c.43.703.91 1.003 1.608 1.003.602 0 1.006-.267 1.006-.664 0-.44-.293-.604-1.197-1l-.459-.197c-1.226-.522-2.362-1.248-2.362-3.007 0-1.58 1.212-2.684 2.946-2.684 1.314 0 2.215.509 2.835 1.6l.115.203-1.85 1.19-.128-.228c-.31-.554-.601-.665-.972-.665-.387 0-.636.213-.636.545 0 .38.183.547.984.894zm13.522.323l1.35-3.76h2.314l-3.43 9.533h-.515l-2.08-4.465a9.014 9.014 0 01-.283-.69c-.088.232-.19.487-.287.692l-2.163 4.463h-.51l-3.486-9.533h2.473l1.363 3.76c.087.24.177.533.255.804.092-.28.204-.582.32-.826l1.79-3.816h.496l1.819 3.816c.116.244.227.546.32.826.078-.271.168-.564.254-.804zm6.363 5.5V7.265h2.336v9.454zm9.996-3.177V11.43h4.339v4.237l-.09.074c-.496.408-1.812 1.097-3.594 1.097-3.007 0-5.107-1.993-5.107-4.845 0-2.8 2.026-4.833 4.817-4.833 1.535 0 2.66.433 3.438 1.323l.152.174-1.592 1.576-.173-.178c-.445-.458-.905-.742-1.825-.742-1.43 0-2.428 1.102-2.428 2.68 0 1.643 1.066 2.706 2.718 2.706.547 0 1.118-.094 1.506-.245v-.911zm11.84 0V11.43h4.338v4.237l-.09.074c-.497.408-1.812 1.097-3.594 1.097-3.007 0-5.106-1.993-5.106-4.845 0-2.8 2.026-4.833 4.816-4.833 1.535 0 2.66.433 3.439 1.323l.152.174-1.592 1.576-.173-.178c-.445-.458-.906-.742-1.826-.742-1.43 0-2.428 1.102-2.428 2.68 0 1.643 1.067 2.706 2.718 2.706.548 0 1.118-.094 1.506-.245v-.911zm12.898-6.278h2.442l-3.352 6.291v3.163h-2.349v-3.028l-3.57-6.426h2.606l1.65 3.015c.177.328.363.761.503 1.113.133-.35.31-.779.487-1.11z" transform="translate(3.871)"/></svg>
```

Comment (adamrusted) [reply icons/swiggy.svg:1]: Within Inkscape there's a panel called the 'XML Editor' - from there you can edit the transform code out. It's likely something daft like the 'artboard' being off-center in the program.

**Comment by PeterShaggyNoble** [comment]
Are you still working on this one, @adamalston?

**Comment by adamrusted** [comment]
@PeterShaggyNoble I've just pushed the path I used for comparison. If you could review this one, we could look to getting it merged in 👌

**Comment by hasthamalp** [comment]
it is merged?

**Review by PeterShaggyNoble — APPROVED** [review]
SVG looks perfect to me - nice work, @adamalston & @adamrusted :+1: Going to update the source URL and then merge this in.

**Pull request merged**